GVFS/GVFS.Hooks/Program.cs

```diff
@@ -113,7 +113,8 @@ private static void RemindUpgradeAvailable()
             int reminderFrequency = 10;
             int randomValue = random.Next(0, 100);
 
-            if (randomValue <= reminderFrequency && ProductUpgraderInfo.IsLocalUpgradeAvailable(tracer: null))
+            if (randomValue <= reminderFrequency &&
+                ProductUpgraderInfo.IsLocalUpgradeAvailable(tracer: null, gvfsDataRoot: GVFSHooksPlatform.GetDataRootForGVFS()))

```

Comment:
I did a search and this method is only called here and is passing `null` for the tracer so it really doesn't need to take the tracer if it is never going to be non-null